src/main/java/com/github/onsdigital/babbage/util/URIUtil.java

```diff
@@ -94,7 +94,10 @@ public static String cleanUri(String uriString) {
         if ("/".equals(uriString)) {
             return uriString;
         }
-        uriString = StringUtils.removeEnd(uriString, "/");
+        // Remove multiple trailing slashes
+        while (uriString.endsWith("/")) {
```

Comment:
A while without a break clause always makes me nervous. I would maybe consider adding 
```&& !uri.equal("/")``` 
to the while condition - to prevent ending up with an empty string. If you had `///` for example you would want `/`